scripts/disable_log_statements.sh

```diff
@@ -1,4 +1,9 @@
-#!/bin/sh -e
+#!/bin/bash
+
+set -e
 
 cd pynvim
-find -name '*.py' | xargs -i{} ../scripts/logging_statement_modifier.py {}
+for f in $(find . -name '*.py'); do
+    echo "Processing: $f"
+    ../scripts/logging_statement_modifier.py "$f"
```

Comment:
For reference: this script is a hack to workaround performance issues: https://github.com/neovim/pynvim/issues/250

Ideally we would avoid costly logging in a more architectural way rather than a build-time hack.

Reply:
Right, fully agreed; this seems really hacky. Just quick-fixing the shell script for now because it didn't work in macOS :)